src/Syncs/Adaptive_IBLT.cpp

```diff
@@ -0,0 +1,49 @@
+//
+// Created by ChenXingyu on 6/10/25.
+//
+#include <GenSync/Syncs/Adaptive_IBLT.h>
+#include <GenSync/Aux/Logger.h>
+
+Adaptive_IBLT::Adaptive_IBLT(long numHashes, long numHashCheck, size_t expectedNumEntries, size_t valueSize)
+        : IBLT(numHashes, numHashCheck, expectedNumEntries, valueSize) {}
+
+bool Adaptive_IBLT::listEntries(std::vector<std::pair<NTL::ZZ, NTL::ZZ>> &positive,
```

Comment:
It would be good to comment the internal workings here ... it won't be clear to you in a few months.